### ISSUE TITLE
OpenPane: Duplicate not working

### ISSUE THREAD

**Issue by RyanDuff613** [SketchIO/feedback-summer-2020#93]
When I attempt to duplicate a file in the Load Pane, Sketchpad indicates that the file has been duplicated but the duplicate file does not appear. This is new as I've been able to duplicate in the past. In the video you'll see the success message appear after clicking "Duplicate", the new file is not created. I then open a file and reload the Load Pane and still do not see the newly created file. This occurs across all browsers.
![duplicate](https://user-images.githubusercontent.com/48165321/93654691-9f0bfc80-f9d3-11ea-88eb-d4ad6c5a4d45.gif)

**Comment by mudcube** [comment]
@ryanreece Appears to work locally, but not on the live site, could you push latest version and check whether this is still broken on the live site?

**Comment by mudcube** [comment]
@ryanreece Yes, this happens.  It's only when Google Drive is enabled.  And it only happens for certain files.  I'm not sure the pattern, but some files are duplicatable, and others are not.  I tried logging into various browsers and syncing, then duplicating the affected files:  Still no dice.  So perhaps something is being stored in Google Drive preventing them to be duplicated?

**Comment by mudcube** [comment]
@ryanreece I now have a reproducible case, would this be something that affects DITKI that we could enlist kenny to resolve?  He's just need to sync his Sketchpad with my m@sketch.io Google Drive, to reproduce the error.
